scripts/clients.py

```diff
@@ -35,8 +35,8 @@ def get_all_notifications_from_hsqldb(self) -> list:
 
 class MpClient:
 
-    def __init__(self, mp_url, user="admin", password="admin", client_id="ManagementPortalapp",
-                 client_secret="") -> None:
+    def __init__(self, mp_url, user="admin", password="admin", client_secret="", 
+                 client_id="ManagementPortalapp") -> None:
```

Comment:
You don't need to change the order here if you are using named arguments when calling the function.